lib/utils/dotsnyk/types.ts

```diff
@@ -13,4 +13,4 @@ export type Glob = string;
 //     - tests?/* # files directly inside “test” and/or “tests” directories
 //     - tests/** # all files and directories inside “tests”
 
-export type Config = { exclude: { unmanaged: Glob[] } };
+export type Config = { exclude?: { unmanaged?: Glob[] } };
```

Comment:
interesting to have that as optional. since it's possible to create optional types in TS and everything is like an interface, then I wonder what's the purpose of that kind of values. Is it to indicate and inform user that it might exist?

Reply:
Yep, just to guide the developer about the possible values in the type. We will implement a schema for the yaml parser at some point also, being easier to reason about the code/expected data.